auto generated update of versioned plugin documentation with manual fix (#1320)
elastic/logstash-docs: docs/versioned-plugins/inputs/http-v3.5.0.asciidoc

===== `additional_codecs`

  * Value type is {logstash-ref}/configuration-file-structure.html#hash[hash]
  * Default value is `{"application/json"=>"json"}`

Apply specific codecs for specific content types.
The default codec will be applied only after this list is checked
and no codec for the request's content-type is found

[id="{version}-plugins-{type}s-{plugin}-cipher_suites"]
===== `cipher_suites`

  * Value type is {logstash-ref}/configuration-file-structure.html#array[array]
  * Default value is `[TLS_AES_128_GCM_SHA256, TLS_AES_256_GCM_SHA384, TLS_CHACHA20_POLY1305_SHA256, TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256, TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256, TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384, TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384, TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256, TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256]`

The list of ciphers suite to use, listed by priorities.
The default values applies for OpenJDK 11.0.14 and higher, for older versions the list does not include suites

> not supported by the JDK, such as the ChaCha20 family of ciphers.

Reviewer: Rework for subject/verb agreement, comma splice, and and clarity. 

Can we make: 
"...for older versions the list does not include suites not supported by the JDK." 

Something like: 
"... for older versions, the list includes only suites supported by the JDK"

> The list of ciphers suite to use, listed by priorities.
> The default values applies for OpenJDK 11.0.14 and higher, for older versions the list does not include suites
> not supported by the JDK, such as the ChaCha20 family of ciphers.

Reviewer: ```suggestion
This default list applies for OpenJDK 11.0.14 and higher.
For older JDK versions, the default list includes only suites supported by that version.
For example, the ChaCha20 family of ciphers is not supported in older versions.
```
